Documentation/navigation/navigation.html

```diff
@@ -0,0 +1,149 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <link rel="stylesheet" href="../../Stylesheets/main.css" />
+    <script
+      src="https://kit.fontawesome.com/6026902e47.js"
+      crossorigin="anonymous"
+    ></script>
+
+    <link rel="stylesheet" href="alert.css" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Stark navigation</title>
+  </head>
+  <body>
+    <nav class="navigation container">
+      <div class="nav-brand">Stark UI</div>
+      <ul class="list-non-bullet list-align">
+        <li class="list-inline">
+          <a class="links link-active" href="../../index.html">Home</a>
```

Comment:
Just doing a / would route to the base directory. No need to track back using multiple dots.